src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -1388,6 +1391,12 @@ def aks_update(cmd,     # pylint: disable=too-many-statements,too-many-branches,
             tier="Paid"
         )
 
+    if no_uptime_sla:
+        instance.sku = ManagedClusterSKU(
+            name="Basic",
+            tier="Free"
+        )
```

Comment:
Add a check for `uptime_sla` and `no_uptime_sla` cannot be specified at the same time? Similar to the above:
```
    if attach_acr and detach_acr:
        raise CLIError('Cannot specify "--attach-acr" and "--detach-acr" at the same time.')
```

Reply:
Good point! changed.